src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1118,19 +1128,132 @@ map_injective f.injective u v
 
 end path
 
+/-! ## Transferring between graphs -/
+
+namespace walk
+
+variables {G}
+
+/-- The walk `p` transferred to lie in `H` -/
+@[simp] def transfer : Π {u v : V} (p : G.walk u v) (H : simple_graph V)
```

Comment:
Maybe this should be `protected` ?

Reply:
Here are the tradeoffs as I see them:

If it's `protected`,
* then we never have to worry about it conflicting with anything else named `transfer`
* while you need to refer to `walk.transfer`, you likely would write `p.transfer H prf` anyway

If it's not `protected`,
* then we do have to worry about it conflicting with something named `transfer`, but this is not a common name.
* then we can just write `transfer p H prf`

I'll leave it to you!